hopsworks-api/src/main/java/io/hops/hopsworks/api/agent/CondaCommandView.java

```diff
@@ -0,0 +1,269 @@
+  public PythonDepsFacade.CondaInstallType getInstallType() {
+    return installType;
+  }
+  
+  public void setInstallType(PythonDepsFacade.CondaInstallType installType) {
+    this.installType = installType;
+  }
+  
+  @ApiModelProperty(value = "Name of the library to install")
+  public String getLib() {
+    return lib;
+  }
+  
+  public void setLib(String lib) {
+    this.lib = lib;
+  }
+  
+  @ApiModelProperty(value = "Environment exported as a YML file")
+  public String getEnvironmentYml() {
+    return environmentYml;
+  }
+  
+  public void setEnvironmentYml(String environmentYml) {
+    this.environmentYml = environmentYml;
+  }
+  
+  public CondaCommands toCondaCommands() {
+    final CondaCommands cc = new CondaCommands();
+    cc.setOp(op);
+    cc.setUser(user);
+    cc.setProj(project);
+    cc.setId(id);
+    cc.setArg(arg);
+    cc.setStatus(status);
+    cc.setVersion(version);
+    cc.setChannelUrl(channelUrl);
+    cc.setInstallType(installType);
+    cc.setLib(lib);
+    cc.setEnvironmentYml(environmentYml);
+    return cc;
+  }
+  
+  public static class Builder {
+    private PythonDepsFacade.CondaOp op;
+    private String user;
+    private String project;
+    private Integer id;
+    private String arg;
+    private PythonDepsFacade.CondaStatus status;
+    private String version;
+    private String channelUrl;
+    private PythonDepsFacade.CondaInstallType installType;
+    private String lib;
+    private String environmentYml;
+    
+    public Builder() {}
+    
+    public Builder setCondaOp(PythonDepsFacade.CondaOp op) {
+      this.op = op;
+      return this;
+    }
+    
+    public Builder setUser(String user) {
+      this.user = user;
+      return this;
+    }
+    
+    public Builder setProject(String project) {
+      this.project = project;
+      return this;
+    }
+    
+    public Builder setCommandId(Integer id) {
+      this.id = id;
+      return this;
+    }
+    
+    public Builder setArguments(String arguments) {
+      this.arg = arguments;
+      return this;
+    }
+    
+    public Builder setStatus(PythonDepsFacade.CondaStatus status) {
+      this.status = status;
+      return this;
+    }
+    
+    public Builder setVersion(String version) {
+      this.version = version;
+      return this;
+    }
+    
+    public Builder setChannelUrl(String channelUrl) {
+      this.channelUrl = channelUrl;
+      return this;
+    }
+    
+    public Builder setInstallType(PythonDepsFacade.CondaInstallType installType) {
+      this.installType = installType;
+      return this;
+    }
+    
+    public Builder setLib(String lib) {
+      this.lib = lib;
+      return this;
+    }
+    
+    public Builder setEnvironmentYml(String yml) {
+      this.environmentYml = yml;
+      return this;
+    }
+    
+    public CondaCommandView build() {
+      return new CondaCommandView(this);
+    }
+  
+    public Builder reset() {
```

Comment:
What's the purpose of this reset method?

Reply:
To reset the Builder to an initial state. When I use the Builder to transform from DAOs to Views I do NOT construct new Builder for each command, instead I used the same builder but reset it.